components/views/Demo/Demo.tsx

```diff
@@ -1,20 +1,52 @@
-import React, {useState} from 'react'
+import React, { useState, useEffect } from "react";
 
 type Props = {
   children: React.ReactNode;
-}
+};
 
 const Demo: React.FC<Props> = ({ children }) => {
-  const [count, setCount] = useState(0)
+  const [count, setCount] = useState<number>(0);
```

Comment:
Types for primitives are optional

```suggestion
  const [count, setCount] = useState(0);
```

Reply:
removed optional <number>